### ISSUE TITLE
Airbus Ship Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#335]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Airbus Ship Detection
:red_circle: **Aim** : This project will help us in detecting the airbus ship using the dataset given
:red_circle: **Dataset** : https://www.kaggle.com/c/airbus-ship-detection/data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.
- This issue is only for 'SWOC' and 'JWOC' contributors of 'ML-Crate' project.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Nirvik07** [comment]
Not able to download dataset @abhisheks008

**Comment by abhisheks008** [comment]
You need to have a verified Kaggle profile to download the dataset @Nirvik07

**Comment by ImRajans** [comment]
Hello I would like to work on this project. Please assign it to me
Full name : Rajan Singh
GitHub Profile Link : https://github.com/ImRajans
Participant ID (If not, then put NA) : NA
Approach for this Project : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model..
What is your participant role? - Contributor

**Comment by abhisheks008** [comment]
Issue assigned to you @ImRajans. Go ahead!

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@ImRajans

**Comment by bhanupgpt** [comment]
Hello @abhisheks008 , I would like to work on this, can you assign this issue to me.
Full name : Bhanupriya Gupta
GitHub Profile Link : https://github.com/bhanupgpt
Participant ID (If not, then put NA) : NA
Approach for this Project : I plan to perform the exploratory data analysis and then I will try to implement the model using Random Forest, KNN, SVM, etc and compare all the algorithms to find out the best fitting one.
What is your participant role? KWoC

**Comment by abhisheks008** [comment]
Issue assigned to you @bhanupgpt

**Comment by bhavya-06darshnik** [comment]
Full name : Bhavya Kumar
GitHub Profile Link : https://github.com/bhavya-06darshnik
Participant ID (If not, then put NA) : NA

Approach for this Project : I'll do basic exploratory data analysis & then apply model using KNN, SVM, & find the most fitting one, after comparison.
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)


@abhisheks008 please assign me this issue as IWOC2024

**Comment by abhisheks008** [comment]
Hi @bhavya-06darshnik try to focus on the following models for this project. Here are the models you can try out,
1. Random Forest
2. Decision Tree
3. SVM
4. XgBoost
5. Gradient Boosting
6. Logistic Regression
7. MLP Classifier
8. Lasso Regression
9. Ridge Regression

Issue assigned to you @bhavya-06darshnik

**Comment by abhisheks008** [comment]
Unassigned as the open source event ended up.

**Comment by Shiroirose** [comment]
Full name : Sebonti Patra
GitHub Profile Link : https://github.com/Shiroirose
Participant ID (If not, then put NA) : NA
Approach for this Project : I plan to do exploratory data analysis and then use algorithms like Logistic Regression, SVM, Decision Trees, Random Forest ,XGBoost and ANN and do hyperparameter tuning to find the best models and compare using accuracy and ROC scores.
What is your participant role? : SSOC
Please assign this issue to me .

**Comment by divyansh-2707** [comment]
Name-Divyansh Singh
Git Hub-https://github.com/divyansh-2707
Approach for this Project - To create ML models after data analysis and proper visualization
Participant Role-SSOC'24


Please Assign it to me.....@abhishek008

**Comment by abhisheks008** [comment]
What are the models you are planning to implement for this problem statement?
@divyansh-2707

**Comment by tanuj437** [comment]
Full name : Tanuj Saxena
GitHub Profile Link :https://github.com/tanuj437
Participant ID (If not, then put NA) : NA
Approach for this Project : Will try to fit model starting with basic ones like Random Forest, SVM, Decision tree, Logistic regression and if the result isn't good like low accuracy and sensitivity will go with CNN and other images model
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) SSOC'24

**Comment by abhisheks008** [comment]
Assigned @tanuj437
